### ISSUE TITLE
MatplotlibDeprecationWarning since 1.5

### ISSUE THREAD

**Issue by pelegm** [SciTools/cartopy#711]
Since upgrading matplotlib to 1.5.1, the following example code:
```python
import matplotlib.pyplot as plt

import cartopy.crs as ccrs


def main():
    ax = plt.axes(projection=ccrs.Robinson())

    # make the map global rather than have it zoom in to
    # the extents of any plotted data
    ax.set_global()

    ax.stock_img()
    ax.coastlines()

    plt.plot(-0.08, 51.53, 'o', transform=ccrs.PlateCarree())
    plt.plot([-0.08, 132], [51.53, 43.17], transform=ccrs.PlateCarree())
    plt.plot([-0.08, 132], [51.53, 43.17], transform=ccrs.Geodetic())

    plt.show()


if __name__ == '__main__':
    main()
```

outputs the following warning:
```
/usr/local/lib/python2.7/dist-packages/matplotlib/artist.py:221: MatplotlibDeprecationWarning: This has been deprecated in mpl 1.5, please use the
axes property.  A removal date has not been set.
  warnings.warn(_get_axes_msg, mplDeprecation, stacklevel=1)
```

This perhaps worth attention.

**Comment by QuLogic** [comment]
This should be fixed as part of #354.

**Comment by pelson** [comment]
Closed in #354. Thanks @pelegm. This will be available in v0.14
